test-helpers/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "fuel-tx-test-helpers"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+
+[dependencies]
+fuel-crypto = { version = "0.2", default-features = false, features = ["random"] }
+fuel-tx = { path = "../../fuel-tx", default-features = false, features = ["alloc","builder","internals","random"] }
```

Comment:
path based normal deps aren't compatible with cargo publish.

Reply:
This isn't intended to be published

Reply:
yeah I could see this builder being useful downstream for VM testing, it's very similar to the `TestBuilder` I added there while working on the `TR` and `TRO` opcodes.